pkg/plugin/manager/manager.go

```diff
@@ -90,6 +102,38 @@ func (m *manager) RunMigrations() error {
 	return nil
 }
 
+func (m *manager) runPublicMigrations() error {
+	// Setup DB if not set
+	if m.db == nil {
+		setErr := m.setDB()
+		if setErr != nil {
+			return errors.New(fmt.Sprintf("could not open db: %s", setErr.Error()))
+		}
+	}
+
+	goose.SetTableName("public.goose_db_version")
+
+	// Fix the migrations
```

Comment:
Is this accurate?

Reply:
Nope that is a lying death comment. 

I think the two functions that run migrations should actually call each other.